pbf/migrations/0039_add_spreading_rot_renews_the_earth.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models, migrations
+
+def load_spreading_rot(apps, schema_editor):
+    Spirit = apps.get_model("pbf", "Spirit")
+    Card = apps.get_model("pbf", "Card")
+    rot = Spirit(name="Rot")
+    rot.save()
+    Card(name="Land of Deadfall and Rot", cost=1, type=2, spirit=rot, elements="Water,Earth,Plant").save()
+    Card(name="Infesting Spores", cost=2, type=2, spirit=rot, elements="Fire,Plant,Animal").save()
+    Card(name="Worms and Bugs Enrich the Soil", cost=1, type=2, spirit=rot, elements="Moon,Water,Earth,Animal").save()
+    Card(name="Exaltation of the Re-forming Land", cost=0, type=2, spirit=rot, elements="Sun,Moon,Water,Plant").save()
```

Comment:
The [Wiki](https://spiritislandwiki.com/index.php?title=Spreading_Rot_Renews_the_Earth#tab=Unique_Power_Cards)'s title tag has this casing:

```suggestion
    Card(name="Exaltation of the Re-Forming Land", cost=0, type=2, spirit=rot, elements="Sun,Moon,Water,Plant").save()
```

Reply:
I looked at the what the rules are for title casing and it looks like you're right! All elements of the hyphenated word should be capitalised. And this is consistent with Gift of Wind-Sped Steps and Flocking Red-Talons. I can't tell you why put Re-forming here initially, but it's abundantly clear that I should change it now.